### DISCUSSION TITLE
[Question][Hw3] Segmentation fault

### DISCUSSION THREAD

**Discussion by kws1207** [prosyslab-classroom/cs348-information-security#527] [Q&A]
I inserted my own instruction while iterating each instructions, and returned llm itself.

Then, make succeed but ./sanitizer test/example1.ll produces an error: Segmentation fault (core dumped).

What can be possible reasons, and is there any tips to do debugging in this case?

Any kind of help would be appreciated.

**Comment by RiceBiscuits** [comment]
Just putting print function at possibly flawed point can be easy way to catch bugs.

**Comment by kws1207** [comment]
Thank you!
